imgui/src/draw_list.rs

```diff
@@ -1186,3 +1191,46 @@ impl<'ui> ImageRounded<'ui> {
         }
     }
 }
+
+#[must_use = "should call .build() to draw the object"]
+pub struct Callback<'ui, F> {
+    draw_list: &'ui DrawListMut<'ui>,
+    callback: F,
+}
+
+impl<'ui, F: FnOnce() + 'static> Callback<'ui, F> {
+    /// Typically constructed by [`DrawListMut::add_callback`]
+    pub fn new(draw_list: &'ui DrawListMut<'_>, callback: F) -> Self {
+        Callback {
+            draw_list,
+            callback,
+        }
+    }
+    /// Call the callback when the window is drawn
```

Comment:
Minor thing but could possibly reword this slightly, maybe something like

> Adds the callback to the draw-list so it will be run when the window is drawn

Mainly the "Call the callback" part in original wording "half-implies" that calling `build` runs the callback

Reply:
Yes, that is right, fixed.